### PR TITLE
should run migrator on single file after correcting problem

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by yanan** [comment]
Test Passed on Jenkins http://ide-build-1:9090/ on this commit 8687c7fe26851f53a1c1b2821454326de8f8e2e2 on build 112

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MarkDoneAction.java [hunk 1]
```diff
@@ -46,6 +52,43 @@ public void run( final Problem problem, final ISelectionProvider provider )
...
+
+            @Override
+            protected IStatus run( IProgressMonitor monitor )
+            {
+                try
+                {
+                    MigrateProjectHandler migrateHandler = new MigrateProjectHandler();
+
+                    Path path = new Path( problem.getFile().getPath() );
+                    IProject[] projects = ProjectUtil.getAllPluginsSDKProjects();
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MarkDoneAction.java]: What if the project is not in a PluginsSDK?  what if it is in a maven project?

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java [hunk 2]
```diff
@@ -329,6 +393,7 @@ public void beginTask( String taskName, int totalWork )
 
         try
         {
+            Thread.sleep( 5000 );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java]: What is this for?

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java [hunk 3]
```diff
@@ -479,4 +543,26 @@ private int isAlreadyExist(
         return index;
     }
 
+    private void clearFileMarkers( File file )
+    {
+        IProject[] projects = ProjectUtil.getAllPluginsSDKProjects();
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java]: what if not in plugins-sdk project?

**Pull request closed without merging**